src/table.cpp

```diff
@@ -263,21 +258,24 @@ void table_t::stop(bool updateable, bool enable_hstore_index,
                     m_target->name.c_str());
             if (hstore_mode != HSTORE_NONE) {
                 m_sql_conn->exec(
-                    "CREATE INDEX ON %1% USING GIN (tags) %2%", m_target->name,
-                    (table_space_index ? "TABLESPACE " + table_space_index.get()
-                                       : ""));
+                    "CREATE INDEX ON {} USING GIN (tags) {}"_format(
+                        m_target->name,
+                        (table_space_index
+                             ? "TABLESPACE " + table_space_index.get()
+                             : "")));
             }
             for (size_t i = 0; i < hstore_columns.size(); ++i) {
-                m_sql_conn->exec("CREATE INDEX ON %1% USING GIN (\"%3%\") %4%",
-                                 m_target->name, i, hstore_columns[i],
-                                 (table_space_index
-                                      ? "TABLESPACE " + table_space_index.get()
-                                      : ""));
+                m_sql_conn->exec(
+                    "CREATE INDEX ON {} USING GIN (\"{}\") {}"_format(
+                        m_target->name, i, hstore_columns[i],
```

Comment:
'i' needs to be removed. There is no %2% in the original anymore. I'm mildly surprised that the tests didn't catch that.

Reply:
With the `_format` this should have been detectable at compile time.

Reply:
It seems this is expected behaviour for fmt. too many parameters isn't considered an error. i have fixed this and did a force push.